Fix DependencyInjectionPluginFramework in build.targets

AdCodicem opened the pull request:
This PR fix the following issue occurring when using the plugin in a .NET framework project :
`error MSB3030: Could not copy the file "C:\Users\<username>\.nuget\packages\solidtoken.specflow.dependencyinjection\0.4.1\lib\net45\SolidToken.SpecFlow.DependencyInjection.SpecFlowPlugin.dll" because it was not found.`

Commit:
Fix DependencyInjectionPluginFramework in build.targets
+semver: fix

AdCodicem commented:
I created my own plugin for using Microsoft DependencyInjection with SpecFlow until I saw yours, so here is a small PR which fix an error which occured on one of my project.
I tested it locally

mbhoek commented:
Thank you for your PR. This makes me realise we need tests in place for both .NET Core and .NET Framework. In order to do that, I would like to recreate the error you described.
Can I ask you what version of .NET Framework you used?

AdCodicem commented:
My project is in .NET framework 4.7.2 (`net472`)

Commit:
Added net472 to Target frameworks in test project
Mutualization of Specflow reference.
Using Specflow 3.3.22-beta (only for test project) to resolve the following issue https://github.com/SpecFlowOSS/SpecFlow/issues/1912

AdCodicem commented:
Hi @mbhoek, 
I updated my PR to include the tests as well.
As explained in the commit comment :
- I used Specflow 3.3.22-beta (only for test project) to resolve the following issue https://github.com/SpecFlowOSS/SpecFlow/issues/1912 
Its an issue which throw an error when MSBuild try to generate the `.feature.cs` files happening with .NET Core 3.1
- I mutualized the Specflow references in the test project for the same reasons. 

I believe the tests remain relevant even with these modifications.
